### PR TITLE
i18n(zh-cn): update 6-islands/3.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

update 6-islands/3.mdx

#### Related issues & labels (optional)

- Closes #
- Suggested label: 
- https://github.com/withastro/docs/pull/8507

### CODE REVIEW

File: src/content/docs/zh-cn/tutorial/6-islands/3.mdx [hunk 1]
```diff
@@ -69,3 +62,30 @@ const textCase = "uppercase";
...
+
+<CardGrid>
+  <LinkCard
+    title="使用内容集合"
+    description="重构教程代码，使用内容集合管理博客文章。"
+    href="/zh-cn/tutorials/add-content-collections/"
+  />
+   <LinkCard
+    title="使用视图过渡动画"
+    description="在教程代码中添加视图过渡动画，自定义页面导航"
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/tutorial/6-islands/3.mdx:78]: 句尾需要补充一个缺失的句号“。”

Comment (Nin3lee) [inline src/content/docs/zh-cn/tutorial/6-islands/3.mdx:78]:
```suggestion
    description="在教程代码中添加视图过渡动画，自定义页面导航。"
```

**Review by Nin3lee — CHANGES_REQUESTED** [review]
78行，添加一处缺失的句号“。”

File: src/content/docs/zh-cn/tutorial/6-islands/3.mdx (hunk 1 above)
Comment (liruifengv) [reply src/content/docs/zh-cn/tutorial/6-islands/3.mdx:78]: Thank you!

**Review by Nin3lee — APPROVED** [review]
LGTM!

**Comment by liruifengv** [comment]
Thank you again @Nin3lee

**Pull request merged**